src/main/jhipster/src/main/webapp/app/germplasm-manager/coding/germplasm-coding-dialog.component.html

```diff
@@ -0,0 +1,143 @@
+<div class="container">
+	<div class="modal-header">
+		<h4 class="modal-title font-weight-bold" id="importLotHeader" jhitranslate="germplasm-coding.title">Specify Codes</h4>
+		<button type="button" class="close" data-dismiss="modal" aria-hidden="true"
+				(click)="close()">&times;
+		</button>
+	</div>
+	<div class="modal-body">
+		<div>
+			<form name="codingForm" #codingForm="ngForm">
+				<div class="form-group">
+					<div class="pl-0 col-sm-12">
+						<span class="mandatory-asterisk"></span> Indicates a mandatory field
+					</div>
+				</div>
+				<div class="form-group row required">
+					<label class="col-sm-5 col-form-label font-weight-bold" jhiTranslate="germplasm-coding.coding.level"></label>
+					<div class="col-sm-7">
+						<div class="form-inline">
+							<div class="form-group">
```

Comment:
Better to use Angular looping for each code name type?

Reply:
We only 3 predefined code names. I don't see the benefit of rendering the options in a loop.